docs/concepts/controllers/statefulsets.md

```diff
@@ -0,0 +1,163 @@
+---
```

Comment:
Add assignees here, so we know when this doc has issues raised against it:

---
assignees:
- bprashanth
- enisoc
- erictune
- foxish
- janetkuo
- kow3ns
- smarterclayton

---